docs/development/extensions-core/druid-pac4j.md

```diff
@@ -38,8 +38,9 @@ druid.auth.authenticator.pac4j.type=pac4j
 ### Properties
 |Property|Description|Default|required|
 |--------|---------------|-----------|-------|--------|
+|`druid.auth.pac4j.cookiePassphrase`|passphrase for encrypting the cookies used to manage authentication session with browser. It can be provided as plaintext string or The [Password Provider](../../operations/password-provider.md).|none|Yes|
+|`druid.auth.pac4j.readTimeout`|Socket connect and read timeout duration used when communicating with authentication server|PT5S|No|
+|`druid.auth.pac4j.enableCustomSslContext`|Whether to use custom SSLContext setup via [simple-client-sslcontext](simple-client-sslcontext.md) extension which must be added to extensions list when this property is set to true.|false|No|
```

Comment:
I think reusing `simple-client-sslcontext` is good for now since the client is the same entity (a Druid server); are there any potential cases where you'd want to use a different truststore or keystore when talking to the auth server here vs. another Druid server?

Reply:
thanks for looking, valid question.
I chose to go this route instead of adding separate config for a branch new ssl context to reduce amount of configuration user has. That said, I see that `ldap` doesn't use druid's common ssl context but builds one separately. I haven't personally seen the use cases  for this separation yet but I am not security expert and could be wrong.
If we come across such use case, then I think we should add support for another SSLContext inside druid core code  .. one  for talking to external auth services (oauth server, ldap  server etc) so  that we don't repeat this thing for all auth extensions that happen to talk to external services.